deploy.bash

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# No failures allowed
+set -e
+
+# The name of the preface for the docker images
+DOCKER_NAME="cnxta/cdr-"
+
+# The IP For the place you're storing this
+DEPLOY_IP=""
```

Comment:
The deploy address IP can also be hostname. 
The IP/hostname and port belong together. 
These values are used for the tag and push to a Docker registry, which is not necessarily where it will be deployed.
Make one variable, something like `DOCKER_REGISTRY`. 
You don't have to define it in this file. Let users sets it in their .profile or .bashrc file like so:
`export DOCKER_REGISTRY=myreg.com:4545`